src/pager/pager.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, ChangeDetectionStrategy, OnChanges, Input, Output, EventEmitter} from 'angular2/core';
+
+@Component({
+  selector: 'ngb-pager',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <nav>
+      <ul class="pager">
+        <li [class.disabled]="!hasPrev()"><a (click)="prev()">Previous</a></li>
+        <li [class.disabled]="!hasNext()"><a (click)="next()">Next</a></li>
+      </ul>
+    </nav>
+    `
+})
+export class NgbPager implements OnChanges {
+  private _currentPage = 0;  // internal state
+  @Input() noOfPages = 0;
+  @Input() page = 0;
+  @Output() pageChange = new EventEmitter();
+
+  prev(): void {
+    if (this.hasPrev()) {
+      this.pageChange.emit(--this._currentPage)
+    }
+  }
+
+  next(): void {
+    if (this.hasNext()) {
+      this.pageChange.emit(++this._currentPage)
+    }
+  }
+
+  hasPrev(): boolean { return this._currentPage > 1; }
+
+  hasNext(): boolean { return this._currentPage < this.noOfPages; }
+
+  ngOnChanges(): void { this._currentPage = Math.max(Math.min(this.page, this.noOfPages), 1) }
```

Comment:
One thing I noticed is that the pagination `page` is allowed to be a number or string - do we have to account for this too with a getter/setter as well since page is an input?